ui/src/app/services/user.service.ts

```diff
@@ -1,24 +1,59 @@
-﻿/*
-* Copyright (c) Universidad Privada Boliviana (UPB) - EUBBC-Digital
-* Adriana Orellana, Angel Zenteno, Alex Villazon, Omar Ormachea
-* MIT License - See LICENSE file in the root directory
-*/
-
+﻿/*
+* Copyright (c) Universidad Privada Boliviana (UPB) - EUBBC-Digital
+* Adriana Orellana, Angel Zenteno, Alex Villazon, Omar Ormachea
+* MIT License - See LICENSE file in the root directory
+*/
+
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
-import { Observable } from 'rxjs';
+import { HttpClient, HttpErrorResponse, HttpHeaders } from '@angular/common/http';
+import { Observable, catchError, of } from 'rxjs';
+import { ToastrService } from 'ngx-toastr';
 import { User } from '../interfaces/user';
 import config from '../config.json';
 
 @Injectable({
   providedIn: 'root',
 })
 export class UserService {
-  private url: string = `${config.api.baseUrl}${config.api.users.me}`;
+  
+
+  private httpOptions = <any>{};
 
-  constructor(private httpClient: HttpClient) {}
+  constructor(private httpClient: HttpClient,private toastr: ToastrService) {
+    this.httpOptions = {

```

Comment:
Is this part necessary?

Reply:
No, it's not necesarry as some not used imports, thanks for the notice.